### DISCUSSION TITLE
Rename 'select'?

### DISCUSSION THREAD

**Discussion by todd-d-desnz** [r-simmer/simmer#282] [Ideas]
https://github.com/r-simmer/simmer/blob/7ad71a770466487ca9ad91e1f0fa6482f41c8253/R/trajectory-activities.R#L397

'select' is a very common function in dplyr. Perhaps it would make sense to rename the simmer function to 'select_resources' to avoid namespace clashes?

**Comment by Enchufa2** [comment]
I know, and I understand it could be a bit annoying. But this function has been there since... 2016, and that's a pretty strong contract with our users. Besides, something like `select_resources` is longer than typing `simmer::select` anyway. So I'm not in favour. Too much potential trouble, limited benefit.
